packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/AddAccountModal/AddAccountModal.tsx

```diff
@@ -70,17 +73,19 @@ export const AddAccountModal = ({ device, onCancel, symbol, noRedirect }: AddAcc
     const networkPinned = !!symbol;
     const preselectedNetwork = symbol && supportedNetworks.find(n => n.symbol === symbol);
     // or in case of only btc is enabled on bitcoin-only firmware
-    const bitcoinNetwork = networksCompatibility.find(n => n.symbol === 'btc');
     const bitcoinOnlyDefaultNetworkSelection =
         device.firmwareType === FirmwareType.BitcoinOnly &&
         supportedMainnets.length === 1 &&
         allTestnetNetworksDisabled
-            ? bitcoinNetwork
+            ? networks.btc
             : undefined;
 
-    const [selectedNetwork, setSelectedNetwork] = useState<NetworkCompatible | undefined>(
+    const [selectedNetwork, setSelectedNetwork] = useState<Network | undefined>(
         preselectedNetwork || bitcoinOnlyDefaultNetworkSelection,
     );
+    const [selectedAccount, setSelectedAccount] = useState<
+        NetworkAccountCollection[number] | undefined
+    >(undefined);
```

Comment:
What is going on here??
UI lets you choose network, then its account.
Previously, the `selectedNetworks` state served both functions, IMHO quite obscure codewise.
(in coin selection, you select a network, but only from the mainnet/testnet ones, then in account selection, you select a _different network object_ from the same set) 

Now, each UI selection has its own state, which is better readable I think.